circuit-benchmarks/src/copy_circuit.rs

```diff
@@ -51,9 +51,10 @@ mod tests {
         ]);
 
         // Create the circuit
-        let block = generate_full_events_block(degree);
+        let mut block = generate_full_events_block(degree);
+        block.circuits_params.max_rws = 10_000;
+        block.circuits_params.max_copy_rows = 10_000;
```

Comment:
I think we have to calculate this to get a number that works. We can fix in the next PR.